shim/injector.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package shim
+
+import (
+	"context"
+
+	ga4gh "github.com/googlegenomics/ga4gh-identity"
+)
+
+// Injector is a ga4gh.Shim that returns a single fixed Identity.
+type Injector struct {
```

Comment:
Injector seems an odd name to me.  Why not say what it does - eg Fixed/Single/Static?